src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/importer/LocationsImporter.kt

```diff
@@ -32,4 +32,6 @@ class LocationsImporter(private val locationRepo: LocationRepository, private va
             }
         }
     }
+
+    fun allImported() = locationRepo.findAll()
```

Comment:
I am a bit unsure on this change and whether this should be here.  Perhaps alternatively return them via the import() method?

Reply:
Got rid of it